Saule/Serialization/ApiError.cs

```diff
@@ -50,6 +73,26 @@ private static string GetRecursiveExceptionMessage(HttpError ex)
             }
 
             return msg;
+        }
+
+        private ExceptionRenderer GetRenderer()
```

Comment:
though, I don't even know if this is the right approach, because my error received in Postman is the same as it was before all these changes